### DISCUSSION TITLE
Visual Studio designer doesn't show content

### DISCUSSION THREAD

**Discussion by maraf** [lepoco/wpfui#399] [Q&A] [answered]
I have created a project using the "WPF UI - MS Store Navigation", but VS designer doesn't show much of the content.
Do you have any idea what could be wrong?

![image](https://user-images.githubusercontent.com/10020471/191291200-48e2fcb5-e33b-4313-90f9-b6b55ccbd425.png)

<details>
    <summary>Visual Studio version</summary>
    <pre>
Microsoft Visual Studio Community 2022
Version 17.3.4
VisualStudio.17.Release/17.3.4+32901.215
Microsoft .NET Framework
Version 4.8.04084

Installed Version: Community

ASP.NET and Web Tools   17.3.376.3011
ASP.NET and Web Tools

Azure App Service Tools v3.0.0   17.3.376.3011
Azure App Service Tools v3.0.0

Azure Functions and Web Jobs Tools   17.3.376.3011
Azure Functions and Web Jobs Tools

C# Tools   4.3.0-3.22423.10+b35bb0baca1071d50cc4f8cf5a1a89e7cc112461
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Neptuo.Productivity.LineDuplication   2.0.0
Productivity tools by Neptuo: Line duplication.

NuGet Package Manager   6.3.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

Razor (ASP.NET Core)   17.0.0.2232702+e1d654e792aa2fe6646a6935bcca80ff0aff4387
Provides languages services for ASP.NET Core Razor.

SQL Server Data Tools   17.0.62207.04100
Microsoft SQL Server Data Tools

Syntax Visualizer   1.0
An extension for visualizing Roslyn SyntaxTrees.

TypeScript Tools   17.0.10701.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   4.3.0-3.22423.10+b35bb0baca1071d50cc4f8cf5a1a89e7cc112461
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools   17.1.0-beta.22363.4+1b94f89d4d1f41f20f9be73c76f4b229d4e49078
Microsoft Visual F# Tools

Visual Studio IntelliCode   2.2
AI-assisted development for Visual Studio.
    </pre>
</details>

**Comment by MikeKatsoulakis** [comment]
What error are you getting?

**Reply by maraf** [reply]
I don't see any errors anywhere. Designer loads without any message. I can click in the designer and it selects correct controls in the XAML below it. It's like everything is ok, only styles are somehow missing..

**Reply by jasmeet21** [reply]
Same here, for me <Window> doesn't show up similar to screenshot above, but content in <Page> do. Still finding the solution.

**Comment by pomianowski** [comment, marked as answer]
The navigation items and the frame content are rendered while the application is running. If you want to edit e.g. DashboardPage, edit it directly. If you want to watch live, you can use the XAML Live Preview in Visual Studio 2022.

Editing window:
![image](https://user-images.githubusercontent.com/13592821/193119521-1176fcbc-0269-4d77-9233-b72dfa5c080a.png)

Editing page:
![image](https://user-images.githubusercontent.com/13592821/193119572-bce5f6ab-4333-4eb8-83a1-9252ed82ce65.png)

Live preview with page and window:
![image](https://user-images.githubusercontent.com/13592821/193120005-4412d3fc-aedd-405d-9d32-5ad5f926649e.png)

**Comment by gjordanp** [comment]
Could someone fix this, I have same problem on VS 2022
![image](https://github.com/lepoco/wpfui/assets/67670790/b3fa9622-7ebd-4964-98be-28854be38906)

But just on designer, when debugging looks OK. Its really anoying
![image](https://github.com/lepoco/wpfui/assets/67670790/cf7e19c8-de30-41c5-9599-a22e09e5aa9b)
